mule-extensions-api-persistence/src/test/resources/extension/list-of-serialized-extension-model.json

```diff
@@ -13,20 +13,22 @@
       "schemaVersion": "",
       "xsdFileName": ""
     },
-    "subTypes": [
-
-    ],
-    "importedTypes": [
-
-    ],
-    "displayModel": {
-
-    },
-    "configurations": [
-
-    ],
+    "subTypes": [],
+    "importedTypes": [],
+    "displayModel": {},
+    "configurations": [],
     "operations": [
       {
+        "errorTypes": [
+          {
+            "type": "SOME_ERROR",
+            "namespace": "ERROR_NAMESPACE",
+            "parent": {
```

Comment:
Do we have some test for the serialization of an errorType which doesn't have a parent?